edx/edx-app-ios
# iOS - Apple Sign in Button

### Description

[LEARNER-7871](https://openedx.atlassian.net/browse/LEARNER-7871)

In this PR I added the Apple sign-in button on the login and registration screen.

### Notes
Please add the following config in iOS.yaml file to enable the Apple sign-in button.

```
APPLE:
    ENABLED: false
```


### How to test this PR
- The Apple Sign in button should show on the login and registration screen after enabling it from the config file.

![Simulator Screen Shot - iPhone 11 - 2020-08-24 at 14 36 16](https://user-images.githubusercontent.com/4454296/91028675-3f5e4380-e617-11ea-8078-ef71dff17237.png)
![Simulator Screen Shot - iPhone 11 - 2020-08-24 at 14 36 21](https://user-images.githubusercontent.com/4454296/91028689-438a6100-e617-11ea-8577-6ddc6f01f555.png)
